tests/kernel/run_ltp.pm

```diff
@@ -260,7 +260,7 @@ sub run {
     assert_script_run('cd /opt/ltp/testcases/bin');
 
     if ($is_network) {
-        script_run('systemctl stop wicked wickedd NetworkManager');
+        script_run('systemctl stop NetworkManager SuSEfirewall2 wicked wickedd');
```

Comment:
@okurz: Well, some time ago you and @richiejp asked for better commit message.
[1]: "I am not sure about the inline comments, because we have the git blame feature amongst other reasons",
[2]: The commit message could have a bit more details, e.g. what issue is fixed, why this needs to change, to which versions of which product it applies, link to a bug or a fate entry, the choices you made, etc.
[3]: "good commit message should answer the "why", not the "what""

Now you want inline comments. I'd personally agree with [1] and prefer it over comments if possible. But if you require it I can add inline comment.

BTW: Is there a style guide for this repository? (I don't see any in this repository). I also take code quality seriously, but I don't want to discuss small issues like this, it's really non-productive.

[1] https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2266#issuecomment-270089453
[2] https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2250#issuecomment-268789913
[3] https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/2385#issuecomment-276652940